server/api/graphql/graphqlServer.js

```diff
@@ -13,6 +20,45 @@ app.use("/graphql", GraphQLHTTP({
   pretty: true
 }));
 
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+app.post("/api/login", Meteor.bindEnvironment((request, response) => {
+  const user = (Meteor.users.find({"emails.address": request.body.email}).fetch());
+  if (!user.length) {
+    return response.send(404).send({message: "User doesn't exist"});
+  }
+  const password = request.body.password;
+  const hash = user[0].services.password.bcrypt;
+  return bcrypt.compare(SHA256(password), hash, (err, res) => {
+    if (err) {
+      return response.status(500).send(err);
+    } else if (!res) {
+      return response.status(401).send({message: "Authentication failed"});
+    }
+    const token = jwt.sign(user[0], secret, {expiresIn: "24h"});
```

Comment:
so what you can do is either use the user.id only or any other user information that is used while authenticating, but userId is usually enough